charts/telemetry/templates/secret.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-client-uuid
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  uuid: {{ .Values.telemetry.uuidgen | b64enc }}
```

Comment:
Newline

Reply:
Please use uuid function to generate uuid, uuid should not be passed from value.yaml:https://helm.sh/docs/chart_template_guide/function_list/#uuid-functions